Mathlib/RingTheory/Nilpotent.lean

```diff
@@ -11,6 +11,7 @@ Authors: Oliver Nash
 import Mathlib.Data.Nat.Choose.Sum
 import Mathlib.Algebra.Algebra.Bilinear
 import Mathlib.RingTheory.Ideal.Operations
+import Mathlib.Algebra.GeomSum
```

Comment:
How much extra does this pull in?

Reply:
Mathlib.Algebra.GeomSum imports the following files:

import Mathlib.Algebra.BigOperators.Order
import Mathlib.Algebra.BigOperators.Ring
import Mathlib.Algebra.BigOperators.Intervals
import Mathlib.Tactic.Abel
import Mathlib.Data.Nat.Parity